src/main/java/uk/gov/ons/ctp/response/casesvc/domain/model/CaseGroup.java

```diff
@@ -61,4 +62,7 @@ public class CaseGroup implements Serializable {
   @Enumerated(EnumType.STRING)
   @Column(name = "status")
   private CaseGroupStatus status;
+
+  @Column(name = "change_state_timestamp")
```

Comment:
Slightly strange name, I would expect it to align with the column? Isn't that status, I also find it more palatable flipping the words, so something like status_change_timestamp

Reply:
Agreed. I've renamed it.